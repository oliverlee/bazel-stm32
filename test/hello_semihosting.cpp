#include <cstdio>

auto main() -> int
{
  // NOLINTNEXTLINE(cppcoreguidelines-pro-type-vararg)
  ::printf("hello semihosting!\n");
}
