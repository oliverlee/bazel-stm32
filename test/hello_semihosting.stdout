QEMU 9.2.4 monitor - type 'help' for more information
(qemu) hello semihosting!
